department-of-veterans-affairs/caseflow
# hotfix/APPEALS-13759: Browser Deprecation of event.path

Resolves [APPEALS-13759](https://vajira.max.gov/browse/APPEALS-13759)

### Description
Added a check to the mousedown event handler to not close the dropdown menu if a user clicks to use the scroll bar

Link to the deprecation page: https://chromestatus.com/feature/5726124632965120

Link to associated caseflow-frontend-toolkit PR: https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit/pull/100

### Acceptance Criteria
- [ ] Menus in caseflow no longer cause a console error and can once again be closed by clicking outside the menu or by using escape key

### Testing Plan
1. Checkout master and run the app
2. Go to any part of caseflow and open the dropdown menu from the header navigation bar
![image](https://user-images.githubusercontent.com/109369527/213261057-0b63fbce-3bae-4e59-8f8c-db5b435bb173.png)
3. Verify that the dropdown menu can not be closed by escape or clicking anywhere on the page
4. Open the browser development console log and verify the errors
![image](https://user-images.githubusercontent.com/109369527/213261471-4596bb14-3aa4-4d51-80b8-3183afa17866.png)

5. Checkout this branch and run the app. You will probably also have to do a yarn install or the corresponding make command client-install
6. Go to any part of caseflow and open the dropdown menu from the header navigation bar
7. Verify that the dropdown menu can once again be closed by escape or clicking anywhere on the page

8. Open the browser development console log and verify the errors are gone
